Digitron1.java

```diff
@@ -0,0 +1,75 @@
+package Domaci1;
+
+import java.util.Scanner;
+
+public class Digitron1 {
+	
+	
+	
+	static Scanner ulaz = new Scanner(System.in);
+	
+
+	
+	
+	
+	 public static void operacijaSabiranje ( int operator1, int operator2) {
+		 int rezultatSabiranja = operator1 + operator2;
+    	 System.out.println("Rezultat sabiranja je: " + rezultatSabiranja);
+	    }
+	 
+	 public static void operacijaOduzimanja ( int operator1, int operator2) {
+		 int rezultatOduzimanja = operator1 - operator2;
+    	 System.out.println("Rezultat oduzimanja je: " + rezultatOduzimanja);
+		 
+	    }
+	 
+	 public static void operacijaMnozenje ( int operator1, int operator2) {
+		 int rezultatMnozenja = operator1 * operator2;
+    	 System.out.println("Rezultat mnozenja je: " + rezultatMnozenja);
+		 
+	    }
+	 
+	 public static void operacijaDeljenje ( int operator1, int operator2) {
+		 int rezultatDeljenja = operator1 / operator2;
+    	 System.out.println("Rezultat deljenja je: " + rezultatDeljenja);
+		 
+	    }
+	
+	 
+	public static void main (String[] args) {
+		
+		System.out.println("Uneti dva cela broja: ");
+		
+		 int operator1 = ulaz.nextInt();
+		 int operator2 = ulaz.nextInt();
+		 
+		 char operacija = 's';

```

Comment:
ne zelimo da char operacija bude hardkodovan, vec kao sto si unosila operatore, tako preko tastature unosis i kod operacije.
sve bi se to takodje vrtelo u petlji, sve dok se za kod operacije ne unese neka vrednost van ovih 4.